plugins/core-plugin/src/main/java/com/google/cloud/teleport/plugin/model/ImageSpecMetadata.java

```diff
@@ -213,14 +212,6 @@ public void setSupportsExactlyOnce(boolean supportsExactlyOnce) {
     this.supportsExactlyOnce = supportsExactlyOnce;
   }
 
-  public String getDefaultStreamingMode() {
-    return defaultStreamingMode;
-  }
-
-  public void setDefaultStreamingMode(String defaultStreamingMode) {
-    this.defaultStreamingMode = defaultStreamingMode;
-  }
-
```

Comment:
Looks like maybe a bad merge? Did you mean to get rid of this?

Reply:
And other streaming mode changes?

Reply:
I did not, good catch